pkgs/data/fonts/line-awesome/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, fetchzip }:
+
+let
+  version = "1.3.0";
+in
+fetchzip {
+  name = "line-awesome-${version}";
+
+  url = "https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/${version}/line-awesome-${version}.zip";
+
+  hash = "sha256:0shxzjpxvicl5l39iqa89r7qry0kxwin3fkya5cgx484h6g300bi";
+
+  postFetch = ''
+    mkdir -p $out/share/fonts
+    unzip -j $downloadedFile ${version}/fonts/\*.ttf -d $out/share/fonts/truetype
```

Comment:
Why is this necessary? Shouldn't `fetchzip` take care of the unpacking?

Reply:
Hm. I modeled the PR after #78409 and other fonts do the same thing, e.g. [yanone-kaffeesatz](https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/fonts/yanone-kaffeesatz/default.nix) or [ubuntu-fonts](https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/fonts/ubuntu-font-family/default.nix). But that doesn't mean it's correct :thinking: 

I think this overrides the `postFetch` defined by [`fetchzip`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/fetchzip/default.nix), which is where the unpacking takes place. So it has to be done manually. But I concur, it's weird.

Reply:
Yeah, it seems very weird. If we're unzipping manually anyway, we might as well use `fetchurl`.

Maybe this should use `mkDerivation` instead, disable the unnecessary phases (mostly build) and do the installation in `installPhase`. I don't have much experience with fonts in nixpkgs though. @marsam, what are your thoughts on this?

Reply:
(Its possible I won't be able to react to emails over the weekend. So if someone else wants to merge, no need to wait for my approval)

Reply:
@puzzlewolf according to the docs, you should just add `unzip` to `nativeBuildInputs`, see: https://nixos.org/nixpkgs/manual/#ssec-unpack-phase .

Reply:
@doronbehar Do you want me to model this after e.g. [nafees](https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/fonts/nafees/default.nix)? It's quite different from e.g. [ubuntu-fonts](https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/fonts/ubuntu-font-family/default.nix), it uses `mkDerivation` instead of `fetchzip`. Do you know of a community-agreed default way of doing this kind of font packages?

Reply:
I can see that we have many fonts installed with `fetchzip` which is a bit unfortunate since this has the disadvantage that it makes it impossible to override these derivations with `<font>.overrideAttrs`. [nafees](https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/fonts/nafees/default.nix) is written great so yes, please.

Reply:
Done, now it uses mkDerivation.